pkg/clients/elasticsearch.go

```diff
@@ -0,0 +1,199 @@
+package clients
```

Comment:
Can we use https://github.com/elastic/go-elasticsearch ?

Reply:
Probably.  I was working on path of least resistance but no reason not to use an official client

Reply:
@richm updated to use `go-elasticsearch`.  Left the fascade around the client to allow mocking and minimize dependency elsewhere in code